src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -885,4 +885,23 @@ Do you want to continue?</value>
   <data name="Make_method_synchronous" xml:space="preserve">
     <value>Make method synchronous.</value>
   </data>
+  <data name="from_0" xml:space="preserve">
+    <value>from {0}</value>
```

Comment:
When this goes into `stabilization`, we'll have to figure out how to get these loc'd, since I think we're past the handoff.